pages/common/s3cmd.md

```diff
@@ -0,0 +1,32 @@
+# s3cmd
+
+> Command line tool and client for uploading, retrieveing and managing data in S3 compatible object storage.
+> More information: <https://s3tools.org/s3cmd>.
+
+- Initial configuration:
+
+`s3cmd --configure`
+
+- List Buckets/Folders/Objects:
```

Comment:
How this can list buckets?

Reply:
`s3cmd ls` by itself will list buckets.
`s3cmd ls s3://{{bucket|path}}`will list contents of that bucket/folder

The s3cmd manpage states:
```
s3cmd ls [s3://BUCKET[/PREFIX]] 
  List objects or buckets
```

I'm not sure how to format the s3 path parameter as optional.

Reply:
your explanation is not clear in this example. you are talking about `ls` without arguments and a specific bucket URI.